### DISCUSSION TITLE
notification created from GM_notification keeps repeating

### DISCUSSION THREAD

**Discussion by zubayer077** [Tampermonkey/tampermonkey#2210] [General]
After the recent update, notification keeps repeating after the first timeout. I created the notification with the following parameters
`{"title":"A title","text":"Here goes the text","timeout":7000,"silent":true, image: <base64 image here>, onclick: function.., ondone: function....}` 
Anyone else can replicate this issue? Found in Google Chrome 129.0.6668.101, Tampermonkey 5.3.1

**Comment by derjanb** [comment]
You're sure `GM_notification` is not called more than once?
Does adding a [`tag`](https://www.tampermonkey.net/documentation.php#api:GM_notification) change the behavior?

**Reply by zubayer077** [reply]
GM_notification is not called more than once, adding a tag doesn't change the behavior

**Comment by derjanb** [comment]
I can reproduce it -> #2211
Thanks.
